### PR TITLE
Add SimpleTranslator from indexer

### PR DESCRIPTION
**Pull request by parkan**

### CODE REVIEW

**Comment by parkan** [comment]
@yusefnapora want to glance over this and push the green button in the morn?

File: mediachain/translation/simple/translator.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @staticmethod
+    def translator_id():
+        return 'SimpleTranslator/0.1'
+
+    @staticmethod
+    def translate(parsed_metadata):
+        simple_json = parsed_metadata
+
+        # extract artist Entity
+        artist_name = simple_json['artist']
```
Comment (yusefnapora) [inline mediachain/translation/simple/translator.py]: maybe wrap this in a try/except in case there's no `artist`?

Comment (autoencoder) [inline mediachain/translation/simple/translator.py]: So as you can see here, basically just copy & pasted the original code, then changed a couple lines to just directly copy in the metadata without remapping the keys at all. Idea is a simple pass-through translator.

Yeah, may be a good idea to add a few more changes other than just that one line.

Maybe detect if there's an artist key, and only build the MC artist entity if there is?

The exact way of getting the artist details may need to be adjusted again due to the new schema changes I made last night, and as we move toward a more finalized schema... E.g. many works in some of our sources list multiple creators. We can discuss it more in slack.

**Comment by yusefnapora** [comment]
yeah, the multiple creators issue is something we need to sort out.  I added a commit that only creates the entity and 'artefactCreatedBy' cell if the 'artist' key exists.  Think I'll merge this now, and we can extend it later as we sort out more schema details

File: mediachain/translation/simple/translator.py (hunk 1 above)
Comment (parkan) [inline mediachain/translation/simple/translator.py]: I'll either rework this directly or make my own version that addresses some of this

**Pull request merged**